contracts/OwnerToken.sol

```diff
@@ -13,18 +13,20 @@ contract OwnerToken is BaseToken {
         string memory _name,
         string memory _symbol,
         string memory _baseTokenURI,
-        string memory _masterName,
-        string memory _masterSymbol,
-        string memory _masterBaseTokenURI,
+        // string memory _masterName,
+        // string memory _masterSymbol,
+        // string memory _masterBaseTokenURI,
+        address _receiver,
         bytes memory _signerPublicKey
     )
         BaseToken(_name, _symbol, 1, false, true, _baseTokenURI, address(this), address(this))
     {
         signerPublicKey = _signerPublicKey;
-        MasterToken masterToken = new MasterToken(_masterName, _masterSymbol, _masterBaseTokenURI, address(this));
-        emit MasterTokenCreated(address(masterToken));
+        // MasterToken masterToken = new MasterToken(_masterName, _masterSymbol, _masterBaseTokenURI, address(this));
+        // emit MasterTokenCreated(address(masterToken));
         address[] memory addresses = new address[](1);
-        addresses[0] = msg.sender;
+        // addresses[0] = msg.sender;
+        addresses[0] = _receiver;
```

Comment:
`msg.sender` will now be the `CommunityTokenDeployer` contract, so we have to pass `receiver` separately, will will be the deployer of the contracts.